package-res/resources/web/prompting/PromptPanel.js

```diff
@@ -343,12 +343,11 @@ define(['cdf/lib/Base', 'cdf/Logger', 'dojo/number', 'dojo/i18n', 'common-ui/uti
            */
           this.destinationId = destinationId;
 
-          if (!paramDefn) {
-            throw 'paramDefn is required';
-          }
           this.paramDefn = paramDefn;
```

Comment:
even better, lets remove this from here and use the setParamDefn method! actually it already does what we need